Dockerfile

```diff
@@ -56,13 +56,15 @@ RUN apk add --no-progress --no-cache \
   && curl -s http://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer \
   && sed -i 's/memory_limit = .*/memory_limit = ${PHP_MEMORY_LIMIT}/' /etc/php8/php.ini \
   && chmod +x /usr/local/bin/composer \
-  && mkdir -p /run/php /flarum/app \
+  && mkdir -p /flarum/app /run/php /var/log/flarum \
   && COMPOSER_CACHE_DIR="/tmp" composer create-project flarum/flarum:$VERSION /flarum/app \
   && composer clear-cache \
-  && rm -rf /flarum/.composer /tmp/* \
+  && rm -rf /flarum/.composer /tmp/* /flarum/app/storage/logs \
+  && cd /flarum/app/storage/ \
+  && ln -s /var/log/flarum logs \
   && setcap CAP_NET_BIND_SERVICE=+eip /usr/sbin/nginx
 
 COPY rootfs /
 RUN chmod +x /usr/local/bin/* /etc/s6.d/*/run /etc/s6.d/.s6-svscan/*
-VOLUME /etc/nginx/flarum /flarum/app/extensions /flarum/app/public/assets /flarum/app/storage/logs
+VOLUME /etc/nginx/flarum /flarum/app/extensions /flarum/app/public/assets /var/log
```

Comment:
I would preferre to open only the flarum logs so /var/log/flarum is better no?

Reply:
> I would preferre to open only the flarum logs so /var/log/flarum is better no?

But the logs for nginx and php-fpm are also collected here...

Reply:
> I would preferre to open only the flarum logs so /var/log/flarum is better no?

I've decided to do it.